docs/modules/ROOT/pages/explanation/decisions/dns-nameing.adoc

```diff
@@ -0,0 +1,39 @@
+= DNS nameing scheme
+
+Problem::
+{product} is a web based service offering.
+As such it is accessible through several domains.
+We need a common understanding how we name things and what domains to use for making it accessible.
+
+Alternatives::
+* In the past, we used different domains for things managed by us and things deployed by users. This due to a concern regarding cross domain cookie access. 
+
+
+Decisison::
+The base domain will be `appuio.cloud`.
+On the third level, we will serve services that are zone independent.
+Services bound to specific zone, will be served through the fourth level where the third level will be the zone's identifier.
```

Comment:
either:
*Services bound to **a** specific zone
*Services bound to specific zone**s**